source/lesson-2-ruby-and-the-command-line.html.md.erb

```diff
@@ -0,0 +1,722 @@
+---
+title: "Lesson 2: Ruby and the Command Line"
+weight: 40
+---
+
+# Lesson 2: Ruby and the Command Line
+
+## Introduction
+
+### Objectives
+
+In this tutorial we are going to look at:
+
+* what a command line is
+* how to use the command line to navigate the file system and run programs
+* what ruby is
+* numbers and strings
+* variables
+* calling and defining methods
+* getting user input
+* doing things multiple times using loops
+* building simple programs
+
+### Goal
+
+By the end of this tutorial you will have built a couple of small ruby
+programs, including one like [this times tables game](https://timestablesgame.richardtowers.repl.run).
+
+## Introduction to the command line
+
+### What is the command line?
+
+The command line is a text interface for your computer. Just like Windows
+Explorer on Windows or Finder on macOs it lets you navigate through the files
+and folders of your computer, but it is completely text based. The command line
+works by typing commands against a prompt, which then gets passed to the
+operating system of the computer that runs these commands.
+
+### How do I access the command line?
+
+To access the command line, we use a terminal emulator, usually called a
+terminal. On macOs you can access the terminal by opening the Terminal
+application from your Applications folder.
+
+The terminal can seem unfamiliar and scary, but it's really just a different
+way of interacting with your computer. So long as you are careful not to run
+any commands you don't understand, you won't do any harm.
+
+### Navigating around in the terminal
+
+Once you opened up your terminal you should see a window like this:
+
+![Screenshot of the macOs terminal app. The text says Last login: Sat May 18 11:08:55 on ttys002 ~$ ](/images/lesson-2/image1.png)
+
+Don't worry if the text in yours is a little different, it doesn't matter.
+
+In our examples, we'll write `$` to represent the prompt, you don't have to
```

Comment:
Not sure I'd know what "the prompt" is as a complete beginner. Does the `$` help readability or is it just one more thing for the person to have to remember?

I found it useful in the Developer Docs when we had to distinguish between VM and host machine (`dev$`/`mac$`) but think it might do more harm than good here.

Reply:
Removed the bit about prompt, and removed the `$ ` from the examples.